### PR TITLE
Fixes the tab functionality in Safari

### PR DESCRIPTION
**Pull request by BryceHayden**

### CODE REVIEW

**Review by BryceHayden — COMMENTED** [review]
I'm leaving a comment for my submit review

File: src/simple/stateless/utils/events.js [hunk 1]
```diff
@@ -31,7 +31,7 @@ export default (key, index, first, last, next, prev, options, optType, typedValu
         update.onFiltered(filtered);
       }
       update.onChange(text);
-      document.activeElement.blur();
+      // document.activeElement.blur();
```
Comment (jmacd3) [inline src/simple/stateless/utils/events.js:34]: I can't think of a reason that we would want to call blur() here. This seems to be the thing that is confusing Safari. I would assume that if we just let tab do it's thing that the blur event naturally gets called when we move off of the activeElement onto the next.

**Comment by bogdan-bodnaruk** [comment]
I think this solution is better https://github.com/deseretdigital-ui/ddm-selecty/pull/20
@jmacd3 You was right about calling `blur`. Everything else was a simple regression. I left my comment why it happened.

**Comment by jmacd3** [comment]
closing in favor of #20

**Pull request closed without merging**